docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -1,113 +1,331 @@
+
+Below we will walk through an example of generating session signatures, the full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/blob/wyatt/get-session-sigs-via-auth-sig/get-session-sigs-via-auth-sig/nodejs).
+
+## Prerequisites
+
+Before continuing with this guide, you should have an understanding of:
+
+- [Session Signatures](../../authentication/session-sigs/intro.md)
+- [Lit Capacity Credits](../../../sdk/capacity-credits.md)
+- [Lit Resources and Abilities](./resources-and-abilities.md)
+
+This guide also has a dependency on the following packages:
+
+- `"@lit-protocol/auth-helpers": "^6.0.0-beta.3"`
+- `"@lit-protocol/constants": "^6.0.0-beta.3"`
+- `"@lit-protocol/lit-node-client": "^6.0.0-beta.3"`
+- `"ethers": "v5"`
+- `"node-localstorage": "^3.0.5"` (Optional dependency depending on whether or not you're executing the Lit code within a browser. Explained further in the [Connecting to the Lit Network](#connecting-to-the-lit-network) section)
+
+## Paying for Usage of the Lit Network
+
+The correct code implementation will depend on whether you're using the free-to-use `cayenne` network, or one of the "paid" networks: `habanero` or `manzano`.
+
+Usage of the `habanero` and `manzano` networks require the use of [Lit Capacity Credits](../../../sdk/capacity-credits.md). Currently, Capacity Credits are paid for using the `testLPX` token and don't require any real-world money. However, in the future you will need to pay real-world money for usage of Lit networks, and `habanero` and `manzano` are the Lit networks were this functionality is being tested and refined.
+
+So, some of the below code snippets will show different implementations based on whether you intend to use the code on `cayenne` or `habanero`/`manzano` networks.
+
+## Specifying an Ethereum Private Key
+
+Our first step will be to specify what Etheruem private key we will use to provide the Authorization Signature for our Session Keys:
+
+```ts
+const ETHEREUM_PRIVATE_KEY = process.env.ETHEREUM_PRIVATE_KEY;
+```
+
+If you're intending to use the `cayenne` network, this can be any Ethereum wallet.
+
+If you're intending to use the `habanero` or `manzano` networks, the corresponding Ethereum address for the private key should have minted Capacity Credits to pay for the request to the Lit network to generate the Session Signatures (please see the [Capacity Credits](../../../sdk/capacity-credits.md) docs for more info on minting Capacity Credits).
+
+## Instantiating an Ethers.js Wallet
+
+Next we'll use our `ETHEREUM_PRIVATE_KEY` constant to instantiate an instance of `ethers.Wallet`:
```

Comment:
```suggestion
Next we'll use our `ETHEREUM_PRIVATE_KEY` constant to instantiate an instance of `ethers.Wallet`:

> **Note:** When working with Ethereum, you can use various blockchain clients or libraries (such as ethers.js, viem, etc.) to manage your private key and sign messages. As long as the chosen class or library has a `signMessage` function, it can be used effectively. 
```